packages/runtime-vapor/__tests__/dom/template.spec.ts

```diff
@@ -40,4 +40,22 @@ describe('api: template', () => {
     expect(nthChild(root, 2)).toBe(root.childNodes[2])
     expect(next(b)).toBe(root.childNodes[2])
   })
+
+  describe('tag closure omission', () => {
```

Comment:
Those tests should be moved to `packages/compiler-vapor/__tests__/abbreviation.spec.ts`